turbinia/workers/__init__.py

```diff
@@ -597,6 +599,8 @@ def run_wrapper(self, evidence):
     Returns:
       A TurbiniaTaskResult object
     """
+    from turbinia.jobs import manager as job_manager  # calls workers job manager
```

Comment:
Is there a reason to import this here and not above where the other imports happen?  A circular dependency maybe?  If so, let's add a comment here about it too.